sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -167,6 +186,11 @@
      */
     private boolean mAutomaticProcessing = true;
 
+    /**
+     * Indicates if the app received a low memory warning in the last session.
+     */
+    private boolean mHasReceivedMemoryWarningInLastSession = false;
```

Comment:
```suggestion
    private boolean mHasReceivedMemoryWarningInLastSession;
```
By convention we don't call out default primitive type values explicitly (false, 0 or null).